wire-ios-sync-engine/Source/SessionManager/UserSessionLoader.swift

```diff
@@ -184,7 +184,7 @@ final class UserSessionLoader {
         )
 
         // Check if this build is blacklisted.
-        if try await isBuildBlacklisted(userSession: userSession) {
```

Comment:
question: I guess this also fix the case where if we can't parse the usersession is not tearDown and so we crash, right?

Reply:
correct